server/routerlicious/packages/lambdas/src/alfred/index.ts

```diff
@@ -46,6 +46,7 @@ import {
 	generateClientId,
 	IRuntimeSignalEnvelope,
 } from "../utils";
+export { IRuntimeSignalEnvelope } from "../utils";
```

Comment:
This can be removed and index can export it directly from "./utils" instead of from "./alfred".